### DISCUSSION TITLE
Login to Bootstrap Studio says cannot validate status

### DISCUSSION THREAD

**Discussion by illuminator3** [community/community#22516] [New to GitHub]
<p>When trying to log into Bootstrap Studio (for the student benefit), I’m getting this:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/a/ca56970fa988b37548f899337141c4fc7afcdaa4.png" data-download-href="/uploads/short-url/sRXXYN1SSWveydZWmbECnU9jNha.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/48388251/181071574-9ccfd450-d444-428a-b7f3-657837b39c5d.png" alt="image" data-base62-sha1="sRXXYN1SSWveydZWmbECnU9jNha" width="690" height="453" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca56970fa988b37548f899337141c4fc7afcdaa4_2_690x453.png, https://aws1.discourse-cdn.com/github/original/3X/c/a/ca56970fa988b37548f899337141c4fc7afcdaa4.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/a/ca56970fa988b37548f899337141c4fc7afcdaa4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca56970fa988b37548f899337141c4fc7afcdaa4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">865×569 64.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
What could be the cause of this? I have an active membership in the program.

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/illuminator3">@illuminator3</a> I recommend using Bootstrap Studio’s <a href="https://forum.bootstrapstudio.io/">forum</a> to ask this query or reach out directly to their support so you can get this resolved.</p>
